File: java_backup/my java/Faceoff/Palindrome.java
class Palindrome
{
    public static void main(int n)
    {
        int i,s=0,k;
        for (i=n;i>0;i=i/10)
        {
            k=i%10;
            s=(s*10)+k;
        }
            System.out.println(s);
        
    }
}

File: 1st_and_2nd_Sem_c_backup/BIN/C Prog/new progs in c/string/palindrome.c
#include<stdio.h>
#include<string.h>
main()
{
	char s[100],temp;
	int i,len=0,c=0;				//c==flag variable
	scanf("%[^\n]",s);
	for(i=0;i<strlen(s)/2;i++)
		{
		if(s[i]==s[strlen(s)-i-1])
			continue;
		else
			c++;
		}
		if(c==0)
			printf("palindronme");
		else
			printf("not palindrome");
}
